### ISSUE TITLE
Keybindings fire in textbox elements

### ISSUE THREAD

**Issue by willsALMANJ** [willsALMANJ/tbkeys#1]
Mousetrap does not know about XUL elements that are not standard HTML, so it does not suppress itself inside of them.

**Comment by ftoledo** [comment]
can this will fixed?

**Comment by willsALMANJ** [comment]
It is fixed for things that are actually the `textbox` class by 1ab9ee094efe2473d17d39c27746e96c28d645be. I have noticed that there are some other text input box elements for which key events still fire. In particular, events do not fire for me for composing emails or entering text in the search box, but they do fire for the quick search box. I want to fix the quick search box issue but I haven't had time to look into it. My workaround for now  is to type my text in the regular search box, select and cut the text with ctrl+A, ctrl+X, and then paste the text into the quick search box.

Have you noticed other text boxes for which events fire?

**Comment by ftoledo** [comment]
yes the quick search textbox and i found that the top rigth "search for addons " on the extension page also.

thanks!

**Comment by willsALMANJ** [comment]
The previous commit fixed `textbox` elements and now b2175069747937d16cdca6d2e5b06997a23460d2 fixes `input` elements like the quick search and addons search fields. I played around in the UI and didn't find any other types of text fields, so hopefully it is good now.
